client/views/chan.tpl

```diff
@@ -1,5 +1,5 @@
 {{#each channels}}
-<div data-id="{{id}}" data-target="#chan-{{id}}" data-title="{{name}}" class="chan {{type}} chan-{{slugify name}}">
+<div data-id="{{id}}" data-target="#chan-{{id}}" data-title="{{name}}" class="chan {{type}} chan-{{slugify name}} chanid-{{id}}">
```

Comment:
Why did you add this?

Reply:
To keep [this function]( https://github.com/creesch/lounge/blob/ab80d640fc988a26d9f1f20cf4cc2309bb5251a3/client/js/lounge.js#L75-L82) from turning into a convoluted spaghetti mess. 
All the information I needed was already available but I needed a reliable way to cross reference it.

Reply:
You can already find it by `data-id`.

Reply:
That doesn't allow me to find it in all instances as jquery `data()` is used after the client is initialized. So when a nickname is changed it might not match the attribute anymore. 

For channels it appears to be relatively safe to use an attribute selector but I could not be certain so opted for including a more reliable selector in both instances.